app/stores/stateless/topbarCategories.js

```diff
@@ -110,13 +110,13 @@ export const SEIZA_STAKE_SIMULATOR: TopbarCategory = registerCategory({
     request.selected.getParent().getNetworkInfo().NetworkId === networks.CardanoMainnet.NetworkId
   ),
 });
-export const CARDANO_DELEGATION: TopbarCategory = registerCategory({
-  className: 'cardanoStake',
-  route: ROUTES.WALLETS.CARDANO_DELEGATION,
-  icon: undefined,
-  label: messages.delegationById,
-  isVisible: request => (
-    asGetStakingKey(request.selected) != null &&
-    isCardanoHaskell(request.selected.getParent().getNetworkInfo())
-  ),
-});
+// export const CARDANO_DELEGATION: TopbarCategory = registerCategory({
```

Comment:
I would delete something rather than comment it out -- if we want to re-use it for later I would leave a comment on why this is commented

Reply:
removed it in this case 👍